plugins/meta/tuning/tuning_test.go

```diff
@@ -487,6 +488,148 @@ var _ = Describe("tuning plugin", func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 
+		It(fmt.Sprintf("[%s] configures and deconfigures tx queue len with ADD/DEL", ver), func() {
+			conf := []byte(fmt.Sprintf(`{
+				"name": "test",
+				"type": "iplink",
+				"cniVersion": "%s",
+				"txQLen": 20000,
```

Comment:
nit: make make this 20000 value a const, which can be re-used throughout the test ?
It is IMO better than consistently repeating this "magic" number.